getMongoData/getMongoData.js

```diff
@@ -0,0 +1,218 @@
+/* global db, tojson, tojsononeline, rs, print, printjson */
+
+/* =================================================
+ * getMongoData.js: MongoDB Config and Schema Report
+ * =================================================
+ *
+ * Copyright MongoDB, Inc, 2015
+ *
+ * Gather MongoDB configuration and schema information.
+ *
+ * To execute on a locally running mongod on default port (27017) without
+ * authentication, run:
+ *
+ *     mongo getMongoData.js > getMongoData.log
+ *
+ * To execute on a remote mongod or mongos with authentication, run:
+ *
+ *     mongo <HOST>:<PORT>/admin -u <ADMIN_USER> getMongoData.js > getMongoData.log
+ *
+ * For details, see
+ * https://github.com/mongodb/support-tools/tree/master/getMongoData.
+ *
+ *
+ * DISCLAIMER
+ *
+ * Please note: all tools/ scripts in this repo are released for use "AS
+ * IS" without any warranties of any kind, including, but not limited to
+ * their installation, use, or performance. We disclaim any and all
+ * warranties, either express or implied, including but not limited to
+ * any warranty of noninfringement, merchantability, and/ or fitness for
+ * a particular purpose. We do not warrant that the technology will
+ * meet your requirements, that the operation thereof will be
+ * uninterrupted or error-free, or that any errors will be corrected.
+ *
+ * Any use of these scripts and tools is at your own risk. There is no
+ * guarantee that they have been through thorough testing in a
+ * comparable environment and we are not responsible for any damage
+ * or data loss incurred with their use.
+ *
+ * You are responsible for reviewing and testing any scripts you run
+ * thoroughly before use in any non-testing environment.
+ *
+ *
+ * LICENSE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var version = "2.5.0";
+
+(function () {
+   "use strict";
+}());
+
+// Copied from Mongo Shell
+function printShardInfo(){
```

Comment:
@ victorhooi asked what's this about, so I'll explain here in case anyone else is wondering and/or has a better way to do it. Essentially we want to capture the output of `sh.status()` (which is a shell helper) and this is what I came up with (a bit hacky and not very future-proof).